src/formats/PDB.cpp

```diff
@@ -261,7 +264,7 @@ Record get_record(const std::string& line) {
                rec == "CAVEAT" || rec == "COMPND" || rec == "EXPDTA" ||
                rec == "KEYWDS" || rec == "OBSLTE" || rec == "SOURCE" ||
                rec == "SPLIT " || rec == "SPRSDE" || rec == "TITLE " ||
-               rec == "JRNL  ") {
+               rec == "JRNL  " || rec.substr(0, 5) == "MODEL" ) {
```

Comment:
You should be able to test for `"MODEL "` (with a space) directly here

Reply:
yeah, I don't know why I did that.